openfe_gromacs/tests/protocols/test_create_systems.py

```diff
@@ -0,0 +1,102 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe-gromacs
+import gufe
+import numpy as np
+import pytest
+from openmm import NonbondedForce
+from openmm.app import GromacsGroFile, GromacsTopFile
+
+from openfe_gromacs.protocols.gromacs_md.md_methods import GromacsMDProtocol
+from openfe_gromacs.protocols.gromacs_utils import create_systems
+
+
+def test_interchange_gromacs(T4_protein_component, tmpdir):
+    solvent = gufe.SolventComponent()
+    smc_components = {}
+    prot_settings = GromacsMDProtocol.default_settings()
+    # The function expects a settings dict
+    settings = {}
```

Comment:
`create_openmm_systems` takes in a settings dict, which in the protocol is created using the `_handle_settings` function. This makes this test not so pretty, as I have to create the dictionary. Should I change the function to instead take in the actual settings objects? Or is this ok?

Reply:
I changed this to instead pass in the individual settings. Alternatively, I was thinking we could also have an `SystemCreationSettings` class that contains the settings necessary to create the OpenMM system to distinguish with the settings that are related to Gromacs. But I'm not sure if that would be confusing to the user.